### ISSUE TITLE
Circular reference

### ISSUE THREAD

**Issue by bobvandevijver** [bolt/core#2810]
Seems like there is a circular reference with the latest version of Bolt

Details
-------

| Question                                              | Answer
|------------------------------------------------------ | ------
| Relevant Bolt Version                                 | 5.0
| Install type                                          |  Composer install
| PHP version                                           | 7.3 |

Reproduction
------------

```
bobvandevijver@webdev:~/website$ bin/console bolt:info -vvv

In CheckCircularReferencesPass.php line 67:

  [Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException]
  Circular reference detected for service "Bolt\Menu\StopwatchBackendMenuBuilder", path: "Bolt\Menu\StopwatchBackendMenuBuilder -> Bolt\Menu\StopwatchBackendMen
  uBuilder".


Exception trace:
  at /var/www/website-v2-dev/vendor/symfony/dependency-injection/Compiler/CheckCircularReferencesPass.php:67
 Symfony\Component\DependencyInjection\Compiler\CheckCircularReferencesPass->checkOutEdges() at /var/www/website-v2-dev/vendor/symfony/dependency-injection/Compiler/CheckCircularReferencesPass.php:70
 Symfony\Component\DependencyInjection\Compiler\CheckCircularReferencesPass->checkOutEdges() at /var/www/website-v2-dev/vendor/symfony/dependency-injection/Compiler/CheckCircularReferencesPass.php:43
 Symfony\Component\DependencyInjection\Compiler\CheckCircularReferencesPass->process() at /var/www/website-v2-dev/vendor/symfony/dependency-injection/Compiler/Compiler.php:91
 Symfony\Component\DependencyInjection\Compiler\Compiler->compile() at /var/www/website-v2-dev/vendor/symfony/dependency-injection/ContainerBuilder.php:744
 Symfony\Component\DependencyInjection\ContainerBuilder->compile() at /var/www/website-v2-dev/vendor/symfony/http-kernel/Kernel.php:545
 Symfony\Component\HttpKernel\Kernel->initializeContainer() at /var/www/website-v2-dev/vendor/symfony/http-kernel/Kernel.php:786
 Symfony\Component\HttpKernel\Kernel->preBoot() at /var/www/website-v2-dev/vendor/symfony/http-kernel/Kernel.php:125
 Symfony\Component\HttpKernel\Kernel->boot() at /var/www/website-v2-dev/vendor/bolt/core/src/Kernel.php:40
 Bolt\Kernel->boot() at /var/www/website-v2-dev/vendor/symfony/framework-bundle/Console/Application.php:168
 Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() at /var/www/website-v2-dev/vendor/symfony/framework-bundle/Console/Application.php:74
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /var/www/website-v2-dev/vendor/symfony/console/Application.php:167
 Symfony\Component\Console\Application->run() at /var/www/website-v2-dev/bin/console:39
```

**Comment by bobvandevijver** [comment]
Seems like it is caused by the latest Symfony release: https://github.com/symfony/symfony/issues/42791.

**Comment by bobdenotter** [comment]
We bumped into this half an hour ago, as well.. Just a bit of bad timing, because when i did the final rounds of monkey-testing this morning, the version that breaks wasn't released yet.. 😅

**Comment by bobdenotter** [comment]
I Ninja-edited a quick fix here: 8bb07252c423b1348ca1d00d904e27a666a38ae0

We'll sort it out properly in the morning.

**Comment by bobdenotter** [comment]
Fixed, now that `symfony/dependency-injection`, version `v5.3.8` has been released.
